src/paths/scripts/create.yaml

```diff
@@ -0,0 +1,13 @@
+summary: Post a new script
+tags:
+  - scripts
+operationId: scriptsCreate
+requestBody:
+  $ref: "../../requestBodies/scripts/createBody.yaml"
```

Comment:
Are requestBodies often reused?
If not, it might be more clear to define the request body inside the yaml itself.